Error: Type com.google.android.gms.measurement.internal.zzip is defined multiple times

This happens when you have the Plugin.FireBase nugets installed.

Steps to reproduce:
1. Create a new MAUI project
2. Install the following nugets

		<PackageReference Include="Plugin.Firebase" Version="3.1.4" />
		<PackageReference Include="Plugin.Firebase.Auth.Google" Version="3.1.2" />
		<PackageReference Include="Plugin.AdMob" Version="2.3.1" />

3. Run the project and you will get the error below.

It looks like there are some conlicting libraries...

1>MSBUILD : java.exe error JAVA0000: Error in obj\Debug\net9.0-android35.0\lp\333\jl\classes.jar:com/google/android/gms/measurement/internal/zzip.class:
1>MSBUILD : java.exe error JAVA0000: Type com.google.android.gms.measurement.internal.zzip is defined multiple times: obj\Debug\net9.0-android35.0\lp\333\jl\classes.jar:com/google/android/gms/measurement/internal/zzip.class, obj\Debug\net9.0-android35.0\lp\334\jl\classes.jar:com/google/android/gms/measurement/internal/zzip.class
1>MSBUILD : java.exe error JAVA0000: Compilation failed
1>MSBUILD : java.exe error JAVA0000: java.lang.RuntimeException: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj\Debug\net9.0-android35.0\lp\333\jl\classes.jar
1>MSBUILD : java.exe error JAVA0000: com/google/android/gms/measurement/internal/zzip.class
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.yu.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:131)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.main(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:5)
1>MSBUILD : java.exe error JAVA0000: Caused by: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: obj\Debug\net9.0-android35.0\lp\333\jl\classes.jar:com/google/android/gms/measurement/internal/zzip.class
1>MSBUILD : java.exe error JAVA0000: 	at Version.fakeStackEntry(Version_8.7.18.java:0)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.T.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:5)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.yu.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:82)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.yu.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:32)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.yu.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:31)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.yu.b(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:2)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:42)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.b(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:13)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:40)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.yu.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:118)
1>MSBUILD : java.exe error JAVA0000: 	... 1 more
1>MSBUILD : java.exe error JAVA0000: Caused by: com.android.tools.r8.internal.g: Type com.google.android.gms.measurement.internal.zzip is defined multiple times: obj\Debug\net9.0-android35.0\lp\333\jl\classes.jar:com/google/android/gms/measurement/internal/zzip.class, obj\Debug\net9.0-android35.0\lp\334\jl\classes.jar:com/google/android/gms/measurement/internal/zzip.class
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.bd0.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:21)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.Z50.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:54)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.Z50.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:10)
1>MSBUILD : java.exe error JAVA0000: 	at java.base/java.util.concurrent.ConcurrentHashMap.merge(ConcurrentHashMap.java:2056)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.Z50.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:6)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.graph.s4$a.d(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:6)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:95)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:44)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:9)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:45)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.d(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:17)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.D8.c(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:71)
1>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.yu.a(R8_8.7.18_f8bee6d6fb926b7ebb3b15bf98f726f9d57471456ea20fce6d17d9a020197688:28)
1>MSBUILD : java.exe error JAVA0000: 	... 6 more
1>MSBUILD : java.exe error JAVA0000: Directory 'obj\Debug\net9.0-android35.0\lp\333' is from 'play-services-measurement-base.aar'.
1>MSBUILD : java.exe error JAVA0000: Directory 'obj\Debug\net9.0-android35.0\lp\334' is from 'play-services-measurement.aar'.
1>Done building project "Test.csproj" -- FAILED.

--- a/docs/Troubleshooting-Android.md
+++ b/docs/Troubleshooting-Android.md
@@ -22,7 +22,15 @@
 </ItemGroup>
 ```
 
-- **If you use Firebase and hit duplicate Google measurement classes** (see closed issue #29): this is usually a dependency graph conflict. After cleaning, you may need to pin or exclude the offending transitive package versions in your app.
+- **If you use Firebase and hit duplicate Google measurement classes** (see issues #29 and #55, e.g. `Type com.google.android.gms.internal.measurement.zzbm is defined multiple times`): the Google "measurement" classes ship in both `Xamarin.GooglePlayServices.Measurement.*` (pulled in by this plugin via `Xamarin.GooglePlayServices.Ads.Lite`) and `Xamarin.Firebase.Analytics` (pulled in by Plugin.Firebase). The build only works when the whole family resolves to the same Google release train. Align them by referencing `Xamarin.Firebase.Analytics` directly in your app at the version matching the resolved `Xamarin.GooglePlayServices.Measurement.Base` (check `obj/project.assets.json`). Verified with Plugin.Firebase 4.2.1 and Xamarin.GooglePlayServices.Ads.Lite 124.0.0.6:
+
+```
+<ItemGroup Condition="'$(TargetFramework)' == 'net10.0-android'">
+	<PackageReference Include="Xamarin.Firebase.Analytics" Version="123.2.0.1" />
+</ItemGroup>
+```
+
+  Note: enabling MultiDex or switching to d8/r8 does **not** fix this — d8/r8 fail on duplicate type definitions rather than deduplicating them.
 
 ### Related issues
 

